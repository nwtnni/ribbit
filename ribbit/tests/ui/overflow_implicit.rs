#[ribbit::pack(size = 64)]
#[derive(Clone)]
struct Foo {
    a: u32,
    b: u64,
}

fn main() {}
