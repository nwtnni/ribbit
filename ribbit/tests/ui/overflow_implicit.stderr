error: Field requires 64 bits at offset 32, but only 32 are available
 --> tests/ui/overflow_implicit.rs:5:5
  |
5 |     b: u64,
  |     ^
